.github/workflows/code_quality.yaml

```diff
@@ -25,6 +25,6 @@ jobs:
       - name: Install deps
         run: pipenv sync --system --dev
       - name: Check formatting
-        run: ruff check . && ruff format --check . && isort --check .
+        run: ruff check . && ruff format --check .
```

Comment:
Is it important to mention this change in `#guild-backend`, so that teams in other pods are aware? Will people need to adjust settings in their code editors? I don't think I will, but not sure what others might need to do.